node/mock/nodesCoordinatorMock.go

```diff
@@ -1,13 +1,13 @@
 package mock
```

Comment:
i've analized this, some of the mocks have specific setup for the local context so that's why i did not push this in this PR yet, but will check more :+1: